apps/meeting/package.json

```diff
@@ -6,11 +6,10 @@
     "@types/react-dom": "^16.9.5",
     "@types/styled-components": "^4.4.2",
     "@types/webpack-env": "^1.15.1",
-    "@typescript-eslint/eslint-plugin": "^2.19.2",
-    "@typescript-eslint/parser": "^2.19.2",
     "amazon-chime-sdk-component-library-react": "^2.12.0",
     "amazon-chime-sdk-js": "^2.24.0",
     "aws-sdk": "^2.731.0",
+    "concurrently": "^6.5.1",
```

Comment:
should this be a devdependency?

Reply:
I did not change any existing dependency placing, I think I only changed for the linting as I added it as a new dependency.

Reply:
Moved this to dev dependency.